torch/csrc/distributed/rpc/request_callback_impl.cpp

```diff
@@ -101,8 +101,8 @@ std::unique_ptr<RpcCommandBase> RequestCallbackImpl::processRpc(
       // TODO: make this asynchronous
       std::shared_ptr<OwnerRRef<IValue>> rref =
           ctx.getOrCreateOwnerRRef<IValue>(srf.rrefId());
-      return c10::guts::make_unique<RRefFetchRet>(
-          RRefFetchRet({rref->getValue()}));
+      std::vector<IValue> ivalues = {rref->getValue()};
```

Comment:
Are you just making the copy explicitly because the line was too long?

Reply:
After addressing conflicts with master, this code no long exists. :)